src/bindgen/ir/enumeration.rs

```diff
@@ -721,6 +721,7 @@ impl Enum {
                     write!(out, "{}enum {}", config.style.cython_def(), tag_name);
                 }
             }
+            _ => unimplemented!(),
```

Comment:
Same, let's at least keep the language listed. But why can't it be reached? Worth a comment.